go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/cosmos/gaia
 go 1.12
 
 require (
-	github.com/btcsuite/btcd v0.0.0-20190629003639-c26ffa870fd8 // indirect
+	github.com/btcsuite/btcd v0.0.0-20190523000118-16327141da8c // indirect
```

Comment:
This seems a sort of downgrade. I'd like to investigate further before approving this